package.json

```diff
@@ -34,6 +34,7 @@
         "prepublishOnly": "npm run build"
     },
     "dependencies": {
+        "@gravity-ui/i18n": "^1.0.0",
```

Comment:
Why it is not a peer anymore?

Reply:
We are no more enforcing it. It's now only for internal use.